doc/api/deprecations.md

```diff
@@ -914,6 +914,16 @@ Type: Runtime
 
 This was never a documented feature.
 
+<a id="DEP01XX"></a>
+### DEP01XX: process.env string coercion
```

Comment:
Just `DEP00XX`

Reply:
We are already at DEP0100 :)

(Unless you meant DEP0XXX…)

Reply:
Nope, I mean DEP00XX. The same tooling that looks for REPLACEME in the docs looks for that pattern also and will flag it on release if it's not set properly

Reply:
Ah, in that case we should fix the tooling also…

Reply:
Agreed, see https://github.com/nodejs/node-core-utils/issues/193